docs/core/deploying/deployment-experiences.md

```diff
@@ -0,0 +1,50 @@
+
+For example, installing the .NET Core 2.2.14 update when you already have the .NET Core 2.2.13 update installed will result in both .NET Core 2.2.13 and .NET Core 2.2.14 being present on the computer. 
+
+As customers installed multiple updates and the updates accumulated on disk, we received strong feedback that the experience was not desirable. Therefore, changes were made to the installer in .NET Core 3.0.
+
+In .NET Core 3.0 and later versions, new major and minor releases continue to install side by side with earlier major and minor releases. **_Patch versions for a previously released minor version will upon successful installation remove any earlier patches for the same minor release._** 
+
+For example, installing 3.0.2 when you already have 3.0.1 installed will result in 3.0.2 replacing 3.0.1.
+
+
+## Roll-Forward
+Starting with Core 2.1, .NET Core updates support patch roll-forward. When you create a self-contained deployment, .NET Core tools automatically include the latest serviced runtime of the .NET Core version that your application targets. The latest version is automatically downloaded from NuGet.org.
+.NET Core 3.0 introduces an opt-in feature that allows your app to roll-forward to the latest major version of .NET Core. Additionally, a new setting has been added to control how roll-forward is applied to your app. Roll-forward can be configured in the following ways:
+* Project file property: RollForward
+* Runtime configuration file property: rollForward
+* Environment variable: DOTNET_ROLL_FORWARD
+* Command-line argument: --roll-forward
+
+One of the following values must be specified. If the setting is omitted, Minor is the default.
+* LatestPatch  
+Roll forward to the highest patch version. This value disables minor version roll-forward.
+* Minor  
+Roll forward to the lowest higher minor version, if requested minor version is missing. If the requested minor version is present, then the LatestPatch policy is used.
+* Major  
+Roll forward to lowest higher major version, and lowest minor version, if requested major version is missing. If the requested major version is present, then the Minor policy is used.
+* LatestMinor  
+Roll forward to highest minor version, even if requested minor version is present. Intended for component hosting scenarios.
+* LatestMajor  
+Roll forward to highest major and highest minor version, even if requested major is present. Intended for component hosting scenarios.
+* Disable  
+Don't roll forward. Only bind to specified version. This policy isn't recommended for general use because it disables the ability to roll forward to the latest patches. This value is only recommended for testing.
```

Comment:
Do we really only recommend this for testing?  It seems like there are a subset of "critical" apps that would want this functionality so that they have complete control over what they are shipping with.

Reply:
The reason we don't recommend this is that servicing including security patches though installed will not take effect, resulting in vulnerabilities in the Core runtime. 

In general Microsoft's position has always been that customer shouldn't block/prevent security updates from applying.